### DISCUSSION TITLE
Increase React Native peer dependency version

### DISCUSSION THREAD

**Discussion by iamacook** [react-native-async-storage/async-storage#614] [Ideas]
Do you plan to increase the React Native peer dependency version to support the latest release, 0.64.1? It is currently causing installation issues for the latest verion of Amplify in particular.

**Comment by tido64** [comment]
Not quite sure I fully understand the issue you're seeing. AsyncStorage does support 0.64.1. `^0.64.0` is equivalent to `0.64.x`:
https://github.com/react-native-async-storage/async-storage/blob/864c64bd0f6414c8959548ec237637a82b7faec3/package.json#L66-L68

**Reply by iamacook** [reply]
Apologies, I've just checked the error again and it is not an issue with this system but Amplify itself relying on an older version.
